src/components/about/Content.js

```diff
@@ -0,0 +1,73 @@
+import { Divider, Flex, Heading, Text, theme } from '@chakra-ui/core';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * @component
+ * @example
+ * return (
+ *   <Content
+ *     heading="MISSION"
+ *     message="Our mission statement..."
+ *     marginTop="25px"
+ *     marginBottom="25px"
+ *     dividerMargin="10px"
+ *   />
+ * )
+ */
+const Content = ({
+  heading,
+  message,
+  marginTop,
+  marginBottom,
+  dividerMargin,
+}) => {
+  return (
+    <Flex
+      direction={['column', 'column', 'column', 'row', 'row']}
+      marginTop={marginTop}
+      marginBottom={marginBottom}
+    >
+      <Heading
+        as="h2"
+        lineHeight="1"
+        verticalAlign="top"
+        fontFamily={theme.fonts['heading-slab']}
+        color={theme.colors['rbb-black-200']}
+        textAlign={['center', 'center', 'center', 'right']}
+        minW="285px"
+        paddingRight={['0', '0', '22px']}
+      >
+        {heading}
+      </Heading>
+      <Divider
```

Comment:
I'll just leave one comment here to mention the margin and padding values below :)